configuration/ldap_config.py

```diff
@@ -15,6 +15,9 @@
 AUTH_LDAP_BIND_DN = os.environ.get('AUTH_LDAP_BIND_DN', '')
 AUTH_LDAP_BIND_PASSWORD = os.environ.get('AUTH_LDAP_BIND_PASSWORD', '')
 
+# Set AUTH_LDAP_USER_DN_TEMPLATE
+AUTH_LDAP_USER_DN_TEMPLATE = os.environ.get('AUTH_LDAP_USER_DN_TEMPLATE', '')
```

Comment:
I think the default value here is `None`? https://django-auth-ldap.readthedocs.io/en/latest/reference.html#auth-ldap-user-dn-template